superset-frontend/packages/superset-ui-core/src/components/Button/index.tsx

```diff
@@ -128,10 +132,13 @@ export function Button(props: ButtonProps) {
         minWidth: cta ? theme.sizeUnit * 36 : undefined,
         minHeight: cta ? theme.sizeUnit * 8 : undefined,
         marginLeft: 0,
+        '& > .ant-btn-icon': {
+          display: 'inline-flex',
```

Comment:
still not big on adding any more styles to `Button`, if anything we should be removing styles here. Are you sure this doesn't mess up any of the hundreds of buttons across the app?